.github/workflows/mmda-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Test with Python ${{ matrix.python-version }}
         run: |
-          pip install -e .[dev,pysbd_predictors,hf_predictors]
+          pip install -e .[dev,test,pysbd_predictors,hf_predictors]
```

Comment:
Could you roll `test` dependencies into `dev` dependencies? is not like someone is going to do dev work on MMDA without running tests 😜